### PR TITLE
refactor(*): update product name in docs

### PR DESCRIPTION
**Pull request by ngjaying**
Signed-off-by: ngjaying <ngjaying@gmail.com>

### CODE REVIEW

**Review by Rory-Z — APPROVED** [review]

**Review by jinfahua — CHANGES_REQUESTED** [review]

File: deploy/chart/kuiper/README.md [hunk 1]
```diff
@@ -22,15 +22,15 @@ Kuiper can be deployed at k3s/k8s cluster through Helm chart. Below takes k3s as
...
-+ With any approaches in below to get Kuiper Helm Chart, here uses `git clone` to get Helm chart.
++ With any approaches in below to get eKuiper Helm Chart, here uses `git clone` to get Helm chart.
 
   + Git clone
 
     ```shell
-    $ git clone https://github.com/emqx/kuiper
-    $ cd kuiper/deploy/chart/kuiper
+    $ git clone https://github.com/lf-edge/ekuiper
+    $ cd ekuiper/deploy/chart/Kuiper
```
[the remark this answers is not in the document]
Comment (ngjaying) [reply deploy/chart/kuiper/README.md:33]: if clone from new repo url https://github.com/lf-edge/ekuiper, then yes

File: deploy/chart/kuiper/README.md [hunk 2]
```diff
@@ -41,27 +41,27 @@ Kuiper can be deployed at k3s/k8s cluster through Helm chart. Below takes k3s as
       $ helm repo add emqx https://repos.emqx.io/charts
       ```
 
-    + Search Kuiper
+    + Search eKuiper
```
[the remark this answers is not in the document]
Comment (ngjaying) [reply deploy/chart/kuiper/README.md:44]: No, the helm chart name is not updated yet

File: deploy/chart/kuiper/README.md [hunk 3]
```diff
@@ -105,16 +105,16 @@ Kuiper can be deployed at k3s/k8s cluster through Helm chart. Below takes k3s as
   Connecting to 127.0.0.1:20498...
   ```
 
-#### Deploy persisted Kuiper
+#### Deploy persisted eKuiper
 
-+ Kuiper realized persisted  `pods` through creating PVC resources and mount `/kuiper/data` directory. **Before deploying Kuiper, user need to create PVC or Storage Classes resource in Kubernetes.**
++ eKuiper realized persisted  `pods` through creating PVC resources and mount `/ekuiper/data` directory. **Before deploying Kuiper, user need to create PVC or Storage Classes resource in Kubernetes.**
```
[the remark this answers is not in the document]
Comment (ngjaying) [reply deploy/chart/kuiper/README.md:110]: yes

File: deploy/chart/kuiper/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ngjaying) [reply deploy/chart/kuiper/README.md]: revert back to Search Kuiper

**Review by jinfahua — APPROVED** [review]

**Pull request merged**